### PR TITLE
feat(usb/host): Update manifest files for USB Host components

### PR DESCRIPTION
**Pull request by roma-jam**
# Checklist

- [x] Component contains License
- [x] Component contains README.md
- [x] Component contains idf_component.yml file with `url` field defined
- [x] Component was added to [upload job](https://github.com/espressif/idf-extra-components/blob/master/.github/workflows/upload_component.yml#L18)
- [x] Component was added to [build job](https://github.com/espressif/idf-extra-components/blob/master/test_app/CMakeLists.txt#L8)
- [x] _Optional:_ Component contains unit tests
- [x] CI passing

# Change description
Changes in the `idf_component.yml` component files: 
- Added `## IDF Component Manager Manifest File` line where absent
- Added `esp32p4` target

### CODE REVIEW

File: usb/usb_host_ch34x_vcp/idf_component.yml [hunk 1]
```diff
@@ -2,8 +2,14 @@
...
 dependencies:
   espressif/usb_host_cdc_acm:
     version: ">=1.0.4,<3.0.0"
     public: true
   idf: ">=4.4"
+
+targets:
+  - esp32s2
+  - esp32s3
+  - esp32p4
```
Comment (tore-espressif) [inline usb/usb_host_ch34x_vcp/idf_component.yml:15]: The reason why I did not add the `targets` to `usb_host_xxx_vcp` components is that the targets requirement are taken from `usb_host_cdc_acm` on which they depend.

I did not want to update all the drivers for a new target, but only the base `usb_host_cdc_acm`.

Does it make sense? can we remove it?

File: usb/usb_host_hid/idf_component.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+## IDF Component Manager Manifest File
 version: "1.0.1"
```
Comment (tore-espressif) [inline usb/usb_host_hid/idf_component.yml:2]: In order for this change to be uploaded to ESP registry, we must bump the `version`.
In this case we can only bump the revision number to `1.0.1~1`. Which means: no changes in code; only a metadata change

**Review by tore-espressif — COMMENTED** [review]
@roma-jam Thanks for the update. I left 2 comments ;)

File: usb/usb_host_ch34x_vcp/idf_component.yml (hunk 1 above)
Comment (roma-jam) [reply usb/usb_host_ch34x_vcp/idf_component.yml:15]: Yes, when I started to add them in the end this popped up in my head.... that that was the reason. 
Agreed.

File: usb/usb_host_hid/idf_component.yml (hunk 2 above)
Comment (roma-jam) [reply usb/usb_host_hid/idf_component.yml:2]: Oh...ok. 
Do we want this line in manifest?

**Comment by roma-jam** [comment]
@tore-espressif , thanks for quick review. 
Regarding the lines - the lines were only in HID and MSC component (because I did HID on the MSC base once) so they had these lines. All other  *yml files don't have them. So I remove the lines and make all *yml files the same.

File: usb/usb_host_vcp/idf_component.yml [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
 url: https://github.com/espressif/idf-extra-components/tree/master/usb/usb_host_vcp
 dependencies:
   espressif/usb_host_cdc_acm:
     version: ">=1.0.4,<3.0.0"
     public: true
   idf: ">=4.4"
+targets:
+  - esp32s2
+  - esp32s3
+  - esp32p4
```
Comment (tore-espressif) [inline usb/usb_host_vcp/idf_component.yml:13]: Same here, the component depends on `usb_host_cdc_acm` so the target requirements are taken from there

**Review by tore-espressif — APPROVED** [review]

**Pull request merged**